lib/ecto/model.ex

```diff
@@ -154,6 +154,11 @@ defmodule Ecto.Model do
     assoc.__struct__.assoc_query(assoc, values)
   end
 
+  def build_embedded(%{__struct__: model} = struct, assoc, attributes \\ %{}) do
```

Comment:
Let's get rid of this function until we figure out the best way to add this. It seems not necessary (so far).